tests/test_validator_integration.py

```
@@ -569,7 +541,18 @@ def test_default_remove_unused_columns_is_true(self):
         """The HF default for remove_unused_columns is True."""
         from gliner.training.trainer import TrainingArguments
 
-        args = TrainingArguments(output_dir="/tmp/_test_ruc")
+        try:
+            args = TrainingArguments(output_dir="/tmp/_test_ruc")
```

Ruff (0.15.1)

[error] Probable insecure usage of temporary file or directory: "/tmp/_test_ruc"

(S108)